rpc/src/server/dev_handler.rs

```diff
@@ -42,14 +42,21 @@ pub async fn dev_blocks(
     };
     let limit = query.get_usize("limit").unwrap_or(50);
 
+    let blocks = base_services::get_blocks(
```

Comment:
@adonagy 
I would refactor get_blocks, to take mapping closure as parameter:
```
map(|(header, json_data)| map_header_and_json_to_full_block_info(header, json_data, &chain_id))
```
```
block_result
                .into_iter()
                .map(|block| block.into())
                .collect::<Vec<SlimBlockData>>()
```

or maybe easier to change get_blocks to return:
```
Vec<(BlockHeaderWithHash, BlockJsonData)>
```
and then map it, because now there is a big overhead:
`Vec<(BlockHeaderWithHash, BlockJsonData)>` -> is mapped to FullBlockInfo (there is a lots of hashes computation) -> and then mapped to SlimBlockData

```